src/_shared/scss/_adverts-books.scss

```diff
@@ -1,3 +1,8 @@
+.adverts--tone-books .button--large {
```

Comment:
This seems to be present, unchanged, in a lot of files - is it worth factoring it out in a common file - maybe `_adverts.scss`?

Reply:
(It doesn't seem to depend on being books, jobs etc.)

Reply:
Definitely that would be much cleaner - I'm going to try and refactor the code in the next couple of weeks, hopefully remove unnecessary files and code...